lighthouse-core/test/audits/errors-in-console-test.js

```diff
@@ -57,10 +59,31 @@ describe('Console error logs audit', () => {
           },
         },
       ],
+      RuntimeExceptions: [{
+        'timestamp': 1506535813608.003,
+        'exceptionDetails': {
+          'url': 'http://www.inmotionhosting.com/support/templates/supportcentertemplate/js/fancybox.js',
```

Comment:
This test has a js failure dependency from this page, if they fix it it will start to fail, remove it as this is covered already from the new smoke test

Reply:
can these be switched to a more generic url instead of a real site? Something like `'http://example.com/fancybox.js'` for the three instances of these should be fine